homeassistant/components/iskra/entity.py

```diff
@@ -0,0 +1,41 @@
+"""Base entity for Iskra devices."""
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import IskraDataUpdateCoordinator
+
+
+class IskraEntity(CoordinatorEntity[IskraDataUpdateCoordinator]):
+    """Representation a base Iskra device."""
+
+    _attr_should_poll = True
```

Comment:
why do we poll?

Reply:
i see Entity class has it by default, removed